### PR TITLE
Small improvements to the 2.7.1 schema text

### PR DESCRIPTION
**Pull request by nevrome**
Creating the PDF in #71 helped me find some rough edges of the schema text. I will collect my suggestions in this PR and ask for review later.

The first change I made was to make the schema compliant with [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119), as suggested by @TCLamnidis. It's a tiny change, but helps to clarify the expressions we used so far.

### CODE REVIEW

**Comment by nevrome** [comment]
I worked through the entire standard to make it structurally more uniform and clear. There should be no changes regarding the meaning of these specifications.

**Comment by nevrome** [comment]
I feel a bit bad for putting this tedious PR on your desk, @stschiff and @TCLamnidis. My thinking is that the Poseidon standard should be clear and reasonably homogeneous across files. Just on coherent document with similar, predictable phrasing and layout.

File: README.md [hunk 1]
```diff
@@ -130,18 +132,16 @@ In addition to these files (and optionally their checksums), the POSEIDON.yml fi
...
 
 - A set of strictly defined core variables (defined by column name) and their possible content are documented here: [janno_columns.tsv](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv)
-- A `.janno` file CAN have all of these core variables, or only a subset of them.
+- A `.janno` file MAY have all of these core variables, or only a subset of them.
 - Only three columns MUST be present to make the file valid: **Poseidon_ID**, **Group_Name** and **Genetic_Sex**
-- Arbitrary columns not defined here CAN be added as long as their column names do not clash with the defined ones.
+- Arbitrary columns not defined here MAY be added as long as their column names do not clash with the defined ones.
 - The column order is irrelevant.
-- If information is unknown or a variable does not apply for a certain sample, then the respective cell(s) can be filled with the NULL value `n/a` or simply an empty string.
+- If information is unknown or a variable does not apply for a certain sample, then the respective cell(s) MAY be filled with the NULL value `n/a` or simply an empty string.
```
Comment (stschiff) [inline README.md]: I would remove `NULL value` here and simply write "filled with `n/a` or simply an empty string. The term `NULL value` sounds like database speak and might confuse people. In worst cases people might literally want to fill "NULL" or so.

File: janno_columns.tsv [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-Relation_To	other individuals (by Poseidon_ID) that are related/identical to this individual, multiple entries separated by ;	String	TRUE	FALSE	FALSE				FALSE	FALSE
+Poseidon_ID	sample identifier as defined by the genetics laboratory (e.g. I1234, BOT001), must fit to the values in the Poseidon package .fam/.ind file, must be unique within one package, if multiple datasets exist for the same individual different Poseidon_IDs are required	String	FALSE	FALSE	FALSE				TRUE	TRUE
+Genetic_Sex	genetic sex of the individual derived from this sample, only F, M or U because the EIGENSTRAT and PLINK formats only support these three, edge cases (e.g. XXY, XYY, X0) are undefined and should be grouped as F, M or U, with a Note added	Char	FALSE	TRUE	FALSE	F;M;U			TRUE	FALSE
+Group_Name	meaningful population/group identifiers for the sample, should follow the geographic-temporal nomenclature proposed by Eisenmann et al. 2018 (https://doi.org/10.1038/s41598-018-31123-z), multiple entries separated by ;, the first value must be equal the group name in the .fam/.ind file	String	TRUE	FALSE	FALSE				TRUE	FALSE
```
Comment (stschiff) [inline janno_columns.tsv:4]: In `Group_Name`: Did you not want to capitalize `should` to `SHOULD` ?

**Review by stschiff — COMMENTED** [review]
Looks all good, I found a few minor things and commented on them.

File: janno_columns.tsv (hunk 2 above)
Comment (nevrome) [reply janno_columns.tsv:4]: I did not apply this unified vocabulary in the field/variable definition files, only in the main schema text.

File: README.md (hunk 1 above)
Comment (nevrome) [reply README.md]: Good point. I changed it as you suggested.

File: janno_columns.tsv (hunk 2 above)
Comment (stschiff) [reply janno_columns.tsv:4]: Ah right. OK then.

**Review by stschiff — APPROVED** [review]
Good to go!

**Pull request merged**